### DISCUSSION TITLE
Refetch after invalidate doesn't respect the enabled false

### DISCUSSION THREAD

**Discussion by juansesierra** [TanStack/query#7730] [Q&A]
In react query v5 I'm facing the same problem described [here](https://github.com/TanStack/query/issues/1965).
After a mutation I'm calling the query invalidation function with refetch type 'all' and it is calling the queryFn even if it's disabled. it is someone else facing the same issue?

**Comment by TkDodo** [comment]
> After a mutation I'm calling the query invalidation function with refetch type 'all' and it is calling the queryFn even if it's disabled.

not possible from the way I see the code. Please show a minimal reproduction
